gatsby-config.js

```diff
@@ -73,7 +77,60 @@ module.exports = {
     },
     "gatsby-plugin-dark-mode",
     "gatsby-transformer-sharp",
-    "gatsby-plugin-sitemap",
+    {
+      resolve: "gatsby-plugin-sitemap",
+      options: {
+        query: `
+        {
+          site {
+            siteMetadata {
+              siteUrl
+            }
+          }
+          allSitePage {
+            nodes {
+              path
+            }
+          }
+          allMarkdownRemark {
+            nodes {
+              fields {
+                slug
+              }
+              frontmatter {
+                date(formatString: "YYYY-MM-DD")
+              }
+            }
+          }
+        }
+      `,
+        resolveSiteUrl: () => siteUrl,
+        serialize: ({ site, allMarkdownRemark }) => {
+          // nodes looks like array but its acutally an object
+          const pages = allMarkdownRemark.nodes.reduce((acc, node) => {
+            acc.push({
+              url: `${site.siteMetadata.siteUrl}${node.fields.slug}`,
+              changefreq: `yearly`,
```

Comment:
doesn't this make updates after publish useless?

Reply:
thought blog posts are not updating they just are there. Could be monthly but with amount of updates we do to them google will flag it "never" anyway